src/main/java/com/mraof/minestuck/command/TransportalizerCommand.java

```diff
@@ -33,7 +33,7 @@
 	public static void register(CommandDispatcher<CommandSourceStack> dispatcher)
 	{
 		dispatcher.register(Commands.literal("tpz").requires(commandSource -> commandSource.hasPermission(2))
-				.then(Commands.argument("code", StringArgumentType.word()).executes(context -> teleport(context.getSource(), Collections.singleton(context.getSource().getEntityOrException()), StringArgumentType.getString(context, "code")))
+				.then(Commands.argument("code", StringArgumentType.word()).executes(context -> teleport(context.getSource(), Collections.singleton(context.getSource().getEntityOrException()), StringArgumentType.getString(context, "code").toUpperCase(Locale.ROOT)))
```

Comment:
I think that it would be better to perform the conversion to upper case inside of the `teleport()` function instead to make it less easy to miss when reading through the code, as well as make less things happen in this part of the code where there's already a bunch going on. Plus, by putting it inside of `teleport()`, it'll also apply to the missed call on line 37.